src/room/InCallView.tsx

```diff
@@ -231,11 +292,19 @@ export function InCallView({
         focused: true,
         isLocal: screenshareFeed.isLocal(),
         presenter: false,
+        connectionState: ConnectionState.CONNECTED, // by definition since the screen shares arrived on the same connection
```

Comment:
What if the connection dropped later?

Reply:
True - have passed through the connection state of the relevant user (although we don't display it for screenshares anyway).